sidechain/consensus/aura/src/lib.rs

```diff
@@ -104,7 +104,7 @@ impl<AuthorityPair, ParentchainBlock, SidechainBlock, Environment, OcallApi, Imp
 
 /// The fraction of total block time we are allowed to be producing the block. So that we have
 /// enough time send create and send the block to fellow validateers.
-pub const BLOCK_PROPOSAL_SLOT_PORTION: f32 = 0.8;
+pub const BLOCK_PROPOSAL_SLOT_PORTION: f32 = 0.7;
```

Comment:
Small reduction in the fraction of the slot time we use for executing calls. When running on CI, I found that we often don't have enough time to broadcast a sidechain block before the next slot starts (resulting in duplicate block numbers and discarded blocks)